crates/astria-conductor/src/executor/mod.rs

```diff
@@ -146,20 +146,18 @@ impl Handle<StateIsInit> {
     #[instrument(skip_all, err)]
     pub(crate) async fn send_firm_block(
         self,
-        block: ReconstructedBlock,
+        block: Box<ReconstructedBlock>,
```

Comment:
If we change the function signature you no longer have to explicilty box/unbox in various places:

```suggestion
        block: impl Into<Box<ReconstructedBlock>>,
```

this makes use of the blanket impl `From<T> for Box<T>` and `From<Box<T>> for Box<T>`.

Reply:
Done in d2338a04.